rpmlb/builder/koji.py

```diff
@@ -43,14 +50,23 @@ def __init__(
         self.epel = koji_epel
 
         #: Format string for the build target
-        self.target_format = koji_target_format
+        if koji_target_format is None:
+            self.target_format = self.DEFAULT_TARGET_FORMAT
+        else:
+            self.target_format = koji_target_format
```

Comment:
Is empty string a valid option? If not, this can be simplified to:

```python
self.target_format = koji_target_format or self.DEFAULT_TARGET_FORMAT
```

If empty string is a valid option, it can be changed to:

```python
self.target_format = koji_target_format if koji_target_format is not None else self.DEFAULT_TARGET_FORMAT
```

Which is a pretty long line, so maybe not that good.